lib/beaker/command.rb

```diff
@@ -73,12 +73,16 @@ def cmd_line host, cmd = @command, env = @environment, pc = @prepend_cmds
       env_string = host.environment_string( env )
       prepend_commands = host.prepend_commands( cmd, pc, :cmd_exe => @cmdexe )
       if host[:platform] =~ /cisco/ && host[:user] != 'root'
-          append_command = '"'
+        append_command = '"'
         cmd = cmd.gsub('"') { '\\"' }
       end
 
       # This will cause things like `puppet -t -v agent` which is maybe bad.
-      cmd_line_array = [env_string, prepend_commands, cmd, options_string, args_string, append_command]
+      if cmd =~ /ntpdate/ && host[:platform] =~ /cisco_nexus/
+        cmd_line_array = [env_string, prepend_commands, cmd, options_string, args_string]
+      else
+        cmd_line_array = [env_string, prepend_commands, cmd, options_string, args_string, append_command]
+      end
```

Comment:
@LuvCurves there would be less duplication if this was re-written like so:
```ruby
cmd_line_array = [env_string, prepend_commands, cmd, options_string, args_string]
cmd_line_array << append_command unless (cmd =~ /ntpdate/ && host[:platform] =~ /cisco_nexus/)
```
or breaking the appending command into a multiline conditional because the test is maybe not the simplest to read. Either way, I'm not the biggest fan of re-writing the array here.